test/support/testformat.py

```diff
@@ -0,0 +1,91 @@
+import lit
```

Comment:
For information, I am working on some LLVM changes that would render this file unnecessary. Basically we would just install Lit and the Lit package would contain exactly the same test format as the one used by libc++.